src/logger/logger.ts

```diff
@@ -64,8 +64,16 @@ export default class Logger {
       );
     };
 
+    const truncateSecret = (data: string): string => {
+      return data.replace(
+        /"client_secret":"[a-zA-Z0-9_+=.@/-]{60,}"/gi, // so far I only saw secrets with 64 characters
```

Comment:
You have a forward slash in the regex `...@/-....`. Is Typescript okay with that? I.e. does it not complain that it should be escaped with a backslash like `\/`?

Reply:
actually no, I guess it's because it's inside ""

Reply:
The JS documentation always has slashes escaped https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp#extracting_subdomain_name_from_url 

Maybe a unit test will validate that it works as expected :P